auto_deepnet/utils/data_utils.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import, division, print_function
+import sys
+import os
+curr_path = os.path.abspath(os.path.dirname(__file__))
+sys.path = [os.path.dirname(os.path.dirname(curr_path)), curr_path] + sys.path
```

Comment:
@ClimbsRocks I wasn't sure if this is the cleanest way to do this. Idea was to make it so that auto_deepnet can be imported as is or such, but this seems a little messy imo.